wordpress-mobile/WordPress-Android
# Issue/9815 add analytics

Fixes #9815 Subtask: Add analytics - Would be great to track if people are using this flow and if they are dropping off at any point

To test:
1. Install the wasabi build.
2. Open the **Logcat** window and make sure the WordPress app is selected
3. Select a site that has domain credit available
4. Make sure that **Tracked: domain_credit_prompt_shown** is in the log
5. Click on the domain registration CTA
6. Make sure that **Tracked: domain_credit_redemption_tapped** is in the log
7. Go through the the Domain Registration flow
8. When you arrived at the DomainRegistrationResultFragment screen, make sure that **Tracked: domain_credit_redemption_success** is in the log
